src/main/kotlin/no/nav/familie/ba/sak/behandling/vilkår/VedtakBegrunnelseSpesifikasjon.kt

```diff
@@ -833,10 +830,12 @@ enum class VedtakBegrunnelseSpesifikasjon(val tittel: String, val erTilgjengelig
             else vilkårForBegrunnelse.singleOrNull()
         }
 
-        fun duOgEllerBarnaFødtFormulering(gjelderSøker: Boolean, barnasFødselsdatoer: String, målform: Målform) =
+        fun List<LocalDate>.tilBrevTekst(): String = Utils.slåSammen(this.sorted().map { it.tilKortString() })
```

Comment:
```suggestion
        fun List<LocalDate>.tilBrevTekst(): String = Utils.slåSammen(this.sorted().map { it.tilKortString() })
        fun List<LocalDate>.gjelderFlereBarn(): Boolean = this.size > 1
```

Så i neste pr vil jeg bruke samme data til å sjekke barn. Tipper kanskje også dette ligner mer på input som etterhvert skal sendes til familie-brev for generering av begrunnelser? Altså at man sender liste av datoer/stringer